pyext/src/rmsd_calculation.py

```diff
@@ -365,6 +365,7 @@ def get_rmfs_coordinates_one_rmf(path, rmf_A, rmf_B,
         print("Opening RMF file:", rmf_file, "with",
               n_frames, "frames")
 
+        n_cores = min(n_cores, n_frames)  # to ensure n_per_core > 0
```

Comment:
To ensure that there is no error when a single-frame RMF file is parsed with the n_cores argument set to a value > 1